data/hai/hai missions.txt

```diff
@@ -1656,3 +1656,127 @@ mission "Strider notification"
+					decline
+				`	"Sure, what do you need me to do?"`
+				`	"What do you mean?"`
+			`	"My name is Likeet, and I'm the director of an independent film production. We need you to bring <bunks> crew members and <tons> of film equipment to <planet>, then bring us back after about a month. We can pay you <payment>, but the immaterial rewards of being involved in an ambitious project such as this will be priceless."`
+			choice
+				`	"Why would you want to film in <planet>? Isn't that an abandoned station?"`
+					to display
+						has "visited planet: Darkrest"
+				`	"What's so special about <planet> for filming?"`
+					to display
+						not "visited planet: Darkrest"
+				`	"Sure, I can take you there."`
+					goto accept
+				`	"I'm not going that way. You'll need to find some other captain."`
+					decline
+			`	"Ah, you must not know about <planet>. There are many tales which swirl around the station, chief among them being the existence of ghosts and monsters haunting its corridors. We'd like to take advantage of the popular image of <planet> in the creation of our horror film by using it as a backdrop. It's sort of the human equivalent of setting your film in a 'haunted' forest. Don't worry, we'll bring enough food to last us for the month."`
+			choice
+				`	"Why do people think it's haunted?"`
+				`	"Alright then, I can take you there."`
+					goto accept
+				`	"I'm not going that way. You'll need to find some other captain."`
+					decline
+			`	"Back when the station was still in use, there was a mysterious malfunction in one of the hatches which released oxygen out the airlock. Hundreds on board suffocated to death. To this day, no one knows what caused it, and legends have grown out of the event."`
+			choice
+				`	"What makes you think I'm parking my ship at a haunted station?"`
+				`	"Alright then, I can take you there."`
+					goto accept
+				`	"I'm not going that way. You'll need to find some other captain."`
+					decline
+			`	Likeet gives a laugh. "It's not actually haunted, of course. Those are just stories. Do you think I'd send an entire film crew and all their expensive equipment to a place that was actually dangerous?"`
+			choice
+				`	"Alright then, I can take you there."`
+				`	"I'm not going that way. You'll need to find some other captain."`
+					decline
+			label accept
+			`	"Excellent! I'll let the crew know where to find you."`
+				accept
+	on visit
+		dialog phrase "generic cargo and passenger on visit"
+	on complete
+		event "hai film crew" 30
+		payment 100000 50
+		conversation
+			`You help your passengers unload all of their film equipment once you arrive on <planet>. The props seem to get more and more elaborate as they are unpacked from your ship. You note the presence of countless rubber limbs, mechanical monsters, and cans of fake blood being carried onto the set. Once you are finished, Likeet approaches you.`
+			choice
+				`	"What's the name of your movie, anyways?"`
```

Comment:
Likeet never tells you the name of the movie. Maybe we'll remove it here so that the player still learns the name later on in the culture convo.
```suggestion
```